src/status_im/ui/screens/popover/views.cljs

```diff
@@ -64,15 +65,16 @@
                                                            :left      0
                                                            :right     0
                                                            :transform [{:translateY bottom-anim-value}]}}
-                                     [react/touchable-highlight {:style {:flex 1} :on-press #(re-frame/dispatch [:hide-popover])}
-                                      [react/view {:flex 1 :align-items :center :justify-content :center}
-                                       [react/view {:background-color :white
-                                                    :border-radius    16
-                                                    :margin           32
-                                                    :shadow-offset    {:width 0 :height 2}
-                                                    :shadow-radius    8
-                                                    :shadow-opacity   1
-                                                    :shadow-color     "rgba(0, 9, 26, 0.12)"}
+                                     [react/touchable-highlight {:style    {:flex 1 :align-items :center :justify-content :center}
```

Comment:
have you checked other popovers? do they work fine?

Reply:
checked in iOS only though.. will be sure to check it in Android